packages/data/src/gameData.ts

```diff
@@ -34,6 +42,21 @@ export function getVariant(variantName: string): Variant {
   return variant;
 }
 
+export function getVariantByID(id: number | string): Variant {
```

Comment:
change function signature to only take a number, and then use an ID index that we create on init